### ISSUE TITLE
Error with 2023.11.1

### ISSUE THREAD

**Issue by Natim** [mdomke/schwifty#170]
I encountered this error with the last release:

```
   File "~/python3.11/site-packages/schwifty/__init__.py", line 6, in <module>
    from schwifty.bic import BIC
  File "~/python3.11/site-packages/schwifty/bic.py", line 444, in <module>
    registry.build_index("bank", "bic", key="bic", accumulate=True)
  File "~/python3.11/site-packages/schwifty/registry.py", line 85, in build_index
    base = get(base_name)
           ^^^^^^^^^^^^^^
  File "~/python3.11/site-packages/schwifty/registry.py", line 55, in get
    chunk = json.load(fp)
            ^^^^^^^^^^^^^
  File "~/python3.11/json/__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "~/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by Natim** [comment]
Fixed with 2023.11.2

**Comment by mdomke** [comment]
@Natim I made a mistake when creating the release, because I accidentally included an empty bank registry file. As you wrote it is fixed in `2023.11.2` and in `2023.11.1.post0`. Thanks for reporting though.

**Comment by Natim** [comment]
Thank you for the quick fix and release :pray:
